### ISSUE TITLE
Error while setting up the repo in VS-Code using metals

### ISSUE THREAD

**Issue by kanishk779** [bitcoin-s/bitcoin-s#5260]
**Describe the bug**
Not able to setup the repo in VS-Code using metals

**To Reproduce**
1. Clone the repo locally.
2. Open the folder in VS-Code, note the metals should be already be installed.
3. Import the build.

**Expected behavior**
Setting up the project in VS-code using metals.

**Logs**
```
2023.10.14 19:58:31 ERROR /Users/amankashyap/scalaProjects/bitcoin-s/bitcoin-s-docs/docs.sbt:36: error: type mismatch;
2023.10.14 19:58:31 ERROR  found   : None.type
2023.10.14 19:58:31 ERROR  required: sbt.Result[Option[sbt.File]]
2023.10.14 19:58:31 ERROR     (which expands to)  sbt.Result[Option[java.io.File]]
2023.10.14 19:58:31 ERROR Test / bloopGenerate := None
2023.10.14 19:58:31 ERROR                         ^
2023.10.14 19:58:31 INFO  [error] Type error in expression
2023.10.14 19:58:31 INFO  [warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
2023.10.14 19:58:31 INFO  time: ran 'sbt bloopInstall' in 59s
2023.10.14 19:58:31 ERROR sbt command failed: /Users/amankashyap/Library/Caches/Coursier/arc/https/cdn.azul.com/zulu/bin/zulu17.28.13-ca-jdk17.0.0-macosx_x64.tar.gz/zulu17.28.13-ca-jdk17.0.0-macosx_x64/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /var/folders/7l/7x0qrkd11t108ckj4rzxm6p00000gq/T/metals1355687093307031794/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall
2023.10.14 19:58:58 WARN  no build target for: /Users/amankashyap/scalaProjects/bitcoin-s/bitcoin-s-docs/docs.sbt
2023.10.14 20:35:29 WARN  no build target for: /Users/amankashyap/scalaProjects/bitcoin-s/project/metals.sbt
2023.10.14 20:35:29 WARN  Could not find semantic tokens for: file:///Users/amankashyap/scalaProjects/bitcoin-s/project/metals.sbt
2023.10.14 20:35:29 WARN  Could not find semantic tokens for: file:///Users/amankashyap/scalaProjects/bitcoin-s/project/metals.sbt
2023.10.14 20:35:38 WARN  no build target for: /Users/amankashyap/scalaProjects/bitcoin-s/project/Deps.scala
2023.10.14 20:36:44 WARN  no build target for: /Users/amankashyap/scalaProjects/bitcoin-s/project/TaskKeys.scala

```
**Platform :**
 - OS: macOS
 - Chip: Apple M2 pro
 - Version: Ventura 13.6

**Comment by Christewart** [comment]
Hi! Thank you for the report. I will look into this. I don't usually use VSCode with metals for development rather intellij.

**Comment by kanishk779** [comment]
I tried using IntelliJ and received the same error while importing the SBT build.

**Comment by Christewart** [comment]
Should be fixed with #5226
